lua/DCS-gRPC/methods/coalitions.lua

```diff
@@ -120,6 +120,21 @@ GRPC.methods.addGroup = function(params)
   return GRPC.success({group = GRPC.exporters.group(Group.getByName(template.name))})
 end
 
+GRPC.methods.getStaticObjects = function(params)
+  local result = {}
+  for _, coalitionId in pairs(coalition.side) do
+    if params.coalition == 0 or params.coalition - 1 == coalitionId then -- Decrement for non zero-indexed gRPC enum
+      local staticObjects = coalition.getStaticObjects(coalitionId)
+
+      for _, staticObject in ipairs(staticObjects) do
+        table.insert(result, GRPC.exporters.static(staticObject))
+      end
+    end
```

Comment:
Implemented as such to keep consistency with behavior in `getGroups`:
https://github.com/DCS-gRPC/rust-server/blob/9c35b22d8ac2022c0afe5d9b1fe0cb9853354b28/lua/DCS-gRPC/methods/coalitions.lua#L223-L243